database/factories/ArticleFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Article;
+use App\Models\Category;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class ArticleFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Article::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'title' => $this->faker->sentence(),
+            'text' => collect($this->faker->paragraphs(1))->map(fn($item) => "{$item}")->implode(''),
```

Comment:
Does it really need to be so complex only for some fake text?

Reply:
I was writing multiples <p> tags, but decided to use another approach and didn't refactored this.
Definitely not necessary.